packages/ListBox/src/components/NoResults/NoResults.js

```diff
@@ -11,9 +13,18 @@ const propTypes = {
 export default function NoResults(props) {
   const { label } = props;
   const [state] = useListBox();
+  const I18n = useI18n();
 
   if ((state.isOpen || state.isInline) && state.noResultsFound) {
-    return <sc.NoResults data-pka-anchor="no-results">{label}</sc.NoResults>;
+    return (
+      <>
+        <sc.NoResults data-pka-anchor="no-results">{label}</sc.NoResults>
+        <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} aria-hidden>
+          {/* Using {label} here causes mysterious failures in specs */}
```

Comment:
Weird. Maybe just get rid of the `label` prop then since you're getting it from `i18n` anyway?